docs/achtergronddocumentatie/artefacten/7682.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: page-with-side-nav
 title: B7682 - Afbakening van het concept klantcontact
-date: 31-03-2023
+date: 29-09-2023
 ---
 
 # B7682 - Afbakening van het concept klantcontact
 
 ## Besluit
 
-Een klantcontact is een weergave van het contact met de klant(en).
+Een klantcontact is een weergave van het contact met de klant(en) betreffende één onderwerp.
```

Comment:
Een klantcontact is gedefinieerd als "een contact tussen een klant of een vertegenwoordiger van een klant en de gemeente over één onderwerp." Die vertegenwoordiger ontbreekt hier volgens mij.

Reply:
Aangepast